### PR TITLE
Modules and satellites

### LINKED ISSUE #529: Alignak poller do not load its modules
When defining a poller module, an exception is raised when the poller receives its configuration. The poller cannot unserialize the received modules ...

### LINKED ISSUE #539: Alignak satellites workers queues named as module python_name
The Alignak satellites workers queues are named with the module `python_name` attribute. 

This is a problem with some modules like NRPE booster or SNMP booster because the commands that are attached to a module must be tagged with something like `module_type alignak_module_nrpe_booster` or  `module_type alignak_module_snmp_booster` ...

**Note**: Formerly, satellites used the module `type` property (eg. nrpe_poller, snmp_poller) and we must restore this behavior

### PR DESCRIPTION
**Pull request by mohierf**
Fix #539: satellites queues named as module python_name
Fix #529 and #541: module loading exception in poller and reactionner
Add a daemons with modules launch test
Fix test files rights to 664

-----

Tested also on demo server

### CODE REVIEW

File: alignak/satellite.py [hunk 1]
```diff
@@ -1000,11 +1002,12 @@ def setup_new_conf(self):  # pylint: disable=R0915,R0912
             self.new_modules_conf = []
             for module in mods:
                 # If we already got it, bypass
-                if module.python_name not in self.q_by_mod:
+                if module.get_name() not in self.q_by_mod:
+                    logger.warning("Add module object: %s", module)
```
Comment (ddurieux) [inline alignak/satellite.py]: Prefer use info instead warning

Comment (mohierf) [inline alignak/satellite.py]: You are right ... INFO will be better. I fix this.

File: test/test_daemon_start.py [hunk 2]
```diff
@@ -195,6 +198,26 @@ def test_config_and_start_and_stop(self):
         self.stop_daemon(d)
         self.assertFalse(os.path.exists(d.pidfile))
 
+        # Start as a daemon
```
Comment (Seb-Solon) [inline test/test_daemon_start.py]: comment does not match the real call

Comment (Seb-Solon) [inline test/test_daemon_start.py]: You are a doing two time the same thing actually. The following line are copy paste from the previous ones.

File: test/test_launch_daemons.py [hunk 3]
```diff
@@ -59,6 +59,119 @@ def setUp(self):
...
+                os.remove('/tmp/%sd.pid' % daemon)
+                print("- removed /tmp/%sd.pid" % daemon)
+            if os.path.exists('/tmp/%sd.log' % daemon):
+                os.remove('/tmp/%sd.log' % daemon)
+                print("- removed /tmp/%sd.log" % daemon)
+
+        print("Launching arbiter with bad configuration file...")
+        args = ["../alignak/bin/alignak_arbiter.py",
+                "-V",
+                "-a", "cfg/run_test_launch_daemons/alignak.cfg"]
```
Comment (Seb-Solon) [inline test/test_launch_daemons.py]: ini is not needed with -V ?

File: alignak/satellite.py [hunk 4]
```diff
@@ -83,6 +83,7 @@
 from alignak.daemon import Daemon
 from alignak.stats import statsmgr
 from alignak.check import Check  # pylint: disable=W0611
+from alignak.objects.module import Module  # pylint: disable=W0611
```
Comment (Seb-Solon) [inline alignak/satellite.py]: This import is already imported here : https://github.com/Alignak-monitoring/alignak/blob/develop/alignak/bin/__init__.py. I don't think this was necessary.

File: test/test_daemon_start.py [hunk 5]
```diff
@@ -123,18 +124,19 @@ def get_login_and_group(self, p):
             # so bypass it and keep default value
             return
 
-    def create_daemon(self):
+    def create_daemon(self, is_daemon=False, do_replace=False):
```
Comment (mohierf) [inline test/test_daemon_start.py]: It was to make a test ... the daemon class has this initialization parameter.  Isn't it necessary?

File: test/test_daemon_start.py (hunk 2 above)
Comment (mohierf) [inline test/test_daemon_start.py]: Right, I will fix this

File: test/test_launch_daemons.py (hunk 3 above)
Comment (mohierf) [inline test/test_launch_daemons.py]: Ini is never needed. There are default values for every daemon.

File: alignak/satellite.py (hunk 4 above)
Comment (mohierf) [inline alignak/satellite.py]: Obviously it is not enough ... I thought it was, 

It is not logical, I agree, but after some more tests, this import seems to be needed here :/

**Pull request merged**